Alicerce/Sources/Logging/Log+ConsoleLogDestination.swift

```diff
@@ -0,0 +1,45 @@
+//
+//  Log+ConsoleLogDestination.swift
+//  Alicerce
+//
+//  Created by Meik Schutz on 07/04/2017.
+//  Copyright © 2017 Mindera. All rights reserved.
+//
+
+import Foundation
+
+public extension Log {
+
+    public class ConsoleLogDestination: LogDestination {
+        
+        private static let dispatchQueueLabel = "com.mindera.alicerce.log.destination.console"
+
+        public let queue: Queue
+        public let minLevel: Level
+        public let formatter: LogItemFormatter
+        public private(set) var writtenItems: Int = 0
```

Comment:
As discussed, I think this property (used here and on other `LogDestination`'s) doesn't bring much value and if it's used only for testing purposes we should remove it and find strategies to properly unit test our destinations using dependency injection 😉